tests/unit/lms/services/grading_info_test.py

```diff
@@ -24,32 +24,9 @@ def test_it(self, svc, matching_grading_infos, application_instance):
             matching_grading_infos, key=attrgetter("id")
         )
 
-    @pytest.mark.parametrize(
-        "oauth_consumer_key,context_id,resource_link_id",
-        [
-            (
-                "other_oauth_consumer_key",
-                "matching_context_id",
-                "matching_resource_link_id",
-            ),
-            (
-                "matching_oauth_consumer_key",
-                "other_context_id",
-                "matching_resource_link_id",
-            ),
-            (
-                "matching_oauth_consumer_key",
-                "matching_context_id",
-                "other_resource_link_id",
-            ),
-            ("other_oauth_consumer_key", "other_context_id", "other_resource_link_id"),
-        ],
-    )
-    def test_it_returns_an_empty_list_if_there_are_no_matching_GradingInfos(
-        self, svc, oauth_consumer_key, context_id, resource_link_id
-    ):
```

Comment:
Is something lost here? Previously this was checking each of the items in turn. I'd be interested in your reasoning for removing this, as it does seem like a lot for a little amount of code.

Reply:
Yeah, can't remember what my reasoning was then (if any). 

Kept the same test using application_instance.